Documentation/Helm-Charts/operator-chart.md

```diff
@@ -133,6 +133,10 @@ The following tables lists the configurable parameters of the rook-operator char
 | `admissionController.tolerations`   | Array of tolerations in YAML format which will be added to admission controller deployment.                                                      | `<none>`                                                  |
 | `admissionController.nodeAffinity`  | The node labels for affinity of the admission controller deployment [^1]                                                                         | `<none>`                                                  |
 | `monitoring.enabled`                | Create necessary RBAC rules for Rook to integrate with Prometheus monitoring in the operator namespace. Requires Prometheus to be pre-installed. | `false`                                                   |
+| `csi.csiRBDPluginVolume`            | The volume of the CephCSI RBD plugin DaemonSet [^1]                                                                                | `<none>`                                                  |
+| `csi.csiRBDPluginVolumeMount`       | The volume mounts of the CephCSI RBD plugin DaemonSet [^1]                                                                                | `<none>`                                                  |
+| `csi.csiCephFSPluginVolume`         | The volume of the CephCSI CephFS plugin DaemonSet [^1]                                                                                | `<none>`                                                  |
+| `csi.csiCephFSPluginVolumeMount`    | The volume mounts of the CephCSI CephFS plugin DaemonSet [^1]                                                                                | `<none>`                                                  |
```

Comment:
Don't you think it would be helpful to have some documentation specific on how these options help deploying to NixOS?

Reply:
We dont capture anything related to different arch/OS. This is only to make it configurable. You know how to configure it and ensure it works fine. Can you please help with a Doc PR for NixOS?

Reply:
Yes, if you tell me where you think those docs should go, I'll add them.

Reply:
Perhaps a section could be added on the [Prerequisites page](https://rook.io/docs/rook/latest/Getting-Started/Prerequisites/prerequisites/) (source is [here](https://github.com/rook/rook/blob/master/Documentation/Getting-Started/Prerequisites/prerequisites.md)), but we also don't currently have specific guidance on different platforms. So perhaps we could make that section on the prereqs page general to consider the settings that may be needed on some platforms such as NixOS.